language-server/dm/executionManager.ml

```diff
@@ -51,13 +51,20 @@ type state = {
   initial : Vernacstate.t;
   of_sentence : (sentence_state * feedback_message list) SM.t;
   options : options;
+  doc_id : int; (* unique number used to interface with Coq's Feedback *)
```

Comment:
Given the comment, it should be called `feedback_id` or something like that.

Reply:
A feedback message is `{ doc_id; route_id; span_id; contents }`. So to me `feedback_id` is even more confusing.